File: java/class 3/Question3.java
import java.util.Scanner;

public class Question3{
  public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    System.out.print("Enter the first number: ");
    double num1 = input.nextDouble();
    System.out.print("Enter the second number: ");
    double num2 = input.nextDouble();
    input.close();

    if (num2 == 0){
      throw new ArithmeticException("Cannot divide by zero");
    }

    Integer integer_result = (int) (num1/num2);
    System.out.println(num1 + " divided by " + num2 + " is " + (num1 / num2));
    System.out.println(num1 + " goes into " + num2 + integer_result + " times with a remander of " + (num1 % num2));
    
  }
}

File: python/class 3/question 3.py
num_1=float(input("Enter the first number: "))
num_2=float(input("Enter the second number: "))
print(f"{num_1} divided by {num_2} is {num_1/num_2}")
print(f"{num_1} goes into {num_2} ... {num_1//num_2} times, with a remainder of {num_1%num_2}")




#alternatively you could do 
print(f"{(num_1:=float(input('Enter the first number: ')))} divided by {(num_2:=float(input('Enter the second number: ')))} is {num_1/num_2}\n{num_1} goes into {num_2} ... {num_1//num_2} times, with a remainder of {num_1%num_2}")